drivers/adc/ad7768-1/ad77681.c

```diff
@@ -316,6 +316,54 @@ int32_t ad77681_spi_read_adc_data(struct ad77681_dev *dev,
 	return ret;
 }
 
+/**
+ * Read continuous conversion result from device.
+ * @param dev - The device structure.
+ * @param adc_cont_data - The conversion result data on continuous mode
+ * @return 0 in case of success, negative error code otherwise.
+ */
+int32_t ad77681_spi_read_adc_cont_data(struct ad77681_dev *dev,
+				       uint8_t *adc_cont_data)
+{
+	uint8_t crc_xor_calc_buf[4], buf[6], crc_xor;
+	int32_t ret;
+
+	buf[0] = 0x00;
+	buf[1] = 0x00;
+	buf[2] = 0x00;
+	buf[3] = 0x00;	/* added 2 more array places for max data length read */
+	buf[4] = 0x00;	/* 3 bytes of data (24bit format) + Status bit + CRC */
+
+	ret = spi_write_and_read(dev->spi_desc, buf, dev->data_frame_byte);
+	if (ret < 0)
+		return ret;
+
+	if (dev->crc_sel != AD77681_NO_CRC) {
+		memcpy(crc_xor_calc_buf, buf, dev->data_frame_byte - 1);
```

Comment:
Same here: https://github.com/analogdevicesinc/no-OS/pull/957#discussion_r588077589

Reply:
Done with the edits.